### PR TITLE
add summary comment

### PR DESCRIPTION
**Pull request by tgehrs**
Fix for #442 to create a separate summary comment from the truncated PR comment

### CODE REVIEW

File: coverage_comment/main.py [hunk 1]
```diff
@@ -182,6 +182,24 @@ def process_pr(
             marker=marker,
             subproject_id=config.SUBPROJECT_ID,
         )
+        summary_comment = template.get_comment_markdown(
```
Comment (ewjoachim) [inline coverage_comment/main.py:185]:
```suggestion
        # Same as above except `max_files` is None
        summary_comment = template.get_comment_markdown(
```

**Comment by ewjoachim** [comment]
Can you look at existing tests and tell me if it's easy to add a new one for this case ?

**Comment by tgehrs** [comment]
From my reading of the tests, this one seems to confirm that this functionality should work. It seems that mocks are rarely used, so I'm not sure how else to verify that it is called with None for max_files. If you have a place in mind you would like me to add a test please let me know and I will gladly do my best.

https://github.com/py-cov-action/python-coverage-comment-action/blob/fafc0aee90b594ca6eb2c4689a137b9ea19b6731/tests/unit/test_template.py#L327-L357

**Comment by ewjoachim** [comment]
(Ah sorry, missed you message).

I agree with you, adding a test there will probably mean more mocks, and limited value. Let's go !

**Review by ewjoachim — APPROVED** [review]

**Comment by ewjoachim** [comment]
Released in https://github.com/py-cov-action/python-coverage-comment-action/releases/tag/v3.26 :)

**Pull request merged**